1.1/geo.ttl

```diff
@@ -94,12 +94,12 @@ geosparql:
 	skos:prefLabel "KML Literal"@en ;
 .
 
-:dggsWktLiteral
+:dggsLiteral
 	a rdfs:Datatype ;
 	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
 	skos:definition """A textual serialization of a Discrete Global Grid (DGGS) geometry object."""@en ;
-	skos:example """ "<https://w3id.org/dggs/auspix> OrdinateList (R3234)"^^<http://www.opengis.net/ont/geosparql#dggsWktLiteral>""" ;
-	skos:prefLabel "DGGS Well-Known Text Literal"@en ;
+	skos:example """ "OrdinateList (R3234)"^^<http://www.opengis.net/ont/geosparql#dggsWktLiteral>""" ;
```

Comment:
Should be `^^<http://www.opengis.net/ont/geosparql#dggsLiteral>`

Reply:
Thank you. I  have just fixed the example.